gcs_oauth2_boto_plugin/oauth2_helper.py

```diff
@@ -168,7 +168,12 @@ def OAuth2ApprovalFlow(client, scopes, launch_browser=False):
   # Short delay; webbrowser.open on linux insists on printing out a message
   # which we don't want to run into the prompt for the auth code.
   time.sleep(2)
-  code = input('Enter the authorization code: ')
+
+  try:
```

Comment:
you're basically reverting changes made in the previous 2to3 commit.  you should delete/squash those into one change so the history isn't full of noise.